catalyst_voices/packages/libs/catalyst_cose/test/cose_sign_test.dart

```diff
@@ -2,47 +2,51 @@ import 'dart:convert';
 import 'dart:typed_data';
 
 import 'package:catalyst_cose/catalyst_cose.dart';
-import 'package:convert/convert.dart';
 import 'package:cryptography/cryptography.dart';
 import 'package:test/test.dart';
 
 void main() {
   group(CoseSign, () {
     const uuidV7 = '0193b535-7196-7cd1-84e6-ad9c316cf2d2';
-    late SimplePublicKey publicKey;
     late _SignerVerifier signerVerifier;
 
     setUp(() async {
       // Initialize the Ed25519 algorithm and generate a key pair
       final algorithm = Ed25519();
       final keyPair = await algorithm.newKeyPairFromSeed(List.filled(32, 0));
-      publicKey = await keyPair.extractPublicKey();
       signerVerifier = _SignerVerifier(algorithm, keyPair);
     });
 
     test('sign generates a valid COSE_SIGN structure', () async {
       final coseSign = await CoseSign.sign(
-        protectedHeaders: CoseHeaders.protected(
-          alg: const IntValue(CoseValues.eddsaAlg),
-          kid: hex.encode(publicKey.bytes),
-          contentType: const IntValue(CoseValues.jsonContentType),
-          contentEncoding: const StringValue(CoseValues.brotliContentEncoding),
-          type: const Uuid(uuidV7),
-          id: const Uuid(uuidV7),
-          ver: const Uuid(uuidV7),
-          ref: const SingleReferenceUuid(Uuid(uuidV7)),
-          template: const SingleReferenceUuid(Uuid(uuidV7)),
-          reply: const SingleReferenceUuid(Uuid(uuidV7)),
+        protectedHeaders: const CoseHeaders.protected(
+          contentType: IntValue(CoseValues.jsonContentType),
+          contentEncoding: StringValue(CoseValues.brotliContentEncoding),
+          type: Uuid(uuidV7),
```

Comment:
Document type is UUIDv4, all other UUIDs are v7.

Reply:
fixed